server/config.js.example

```diff
@@ -1,16 +0,0 @@
-module.exports.server = {
```

Comment:
Accidental delete?

Reply:
Yee. I think it's still there though

Reply:
Nope, it disappears when I switch to your branch

Reply:
Forgot to add this. I'll add this in after you guys review this current commit, so in my final commit. Don't resolve this so I remember